docs/docs/development/new-behavior.mdx

```diff
@@ -301,6 +303,111 @@ Comes in the form `static const struct behavior_driver_api <behavior_name>_drive
     For unibody keyboards, all locality values perform the same as `BEHAVIOR_LOCALITY_GLOBAL`.
     :::
 
+##### Behavior Metadata
```

Comment:
```suggestion
##### Behavior metadata
```

Nit, we use [title case](https://zmk.dev/docs/development/contributing/documentation#formatting-and-linting-your-changes) for heading levels 1-3.